src/hats/io/parquet_metadata.py

```diff
@@ -26,34 +26,52 @@ def write_parquet_metadata(
     output_path: str | Path | UPath | None = None,
     create_thumbnail: bool = False,
     thumbnail_threshold: int = 1_000_000,
+    create_metadata: bool = True,
 ):
-    """Generate parquet metadata, using the already-partitioned parquet files
-    for this catalog.
+    """Write Parquet dataset-level metadata files (and optional thumbnail) for a catalog.
 
-    For more information on the general parquet metadata files, and why we write them, see
+    Creates files (relative to the catalog root):
```

Comment:
This table isn't rendered well by sphinx, and the connection to the below paragraphs is awkward. How about:

```
* thumbnail
  * only if create_thumbnail=True
  * <purpose and contents>
* _common_metadata
  * always written
  * <purpose and contents>
* etc...
```

Reply:
Updated based on our conversation earlier today, see rendered docs here: https://hats--605.org.readthedocs.build/en/605/autoapi/hats/io/parquet_metadata/index.html

I mostly copied the descriptions from the IVOA note, and ofc totally open to revisions to writing/formatting/etc